data/mods/gen4/moves.ts

```diff
@@ -1172,6 +1172,12 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			return 100;
 		},
 	},
+	payday: {
+		inherit: true,
+		onHit() {
+			this.add('-fieldactivate', 'move: Pay Day');
```

Comment:
```suggestion
			this.add('-activate', 'move: Pay Day');
```
I looked at the client code and we don't really need to use `fieldactivate`

Reply:
wait nevermind we do this so we don't need to send a specific pokemon lol ignore this